src/uct/api/uct.h

```diff
@@ -693,11 +693,17 @@ ucs_status_t uct_iface_get_address(uct_iface_h iface, uct_iface_addr_t *addr);
  * it does not detect issues such as network mis-configuration or lack of connectivity.
  *
  * @param [in]  iface      Interface to check reachability from.
- * @param [in]  addr       Address to check reachability to.
+ * @param [in]  dev_addr   Device address to check reachability to.
+ * @param [in]  ep_addr    Remote endpoint address to check reachability to.
+ * @param [in]  iface_addr Interface address to check reachability to.
+ *
+ * @note One or more addresses can be NULL depends on interface specific.
+ *       If all addresses are NULL the return value will be 0.
```

Comment:
Sorry, I missed the design call. 
Ok, I will remove `ep_addr`.
I don't understand, what does it mean "we have to pass NULL". We don't have to pass NULL to the function, but we can if we know what we are doing. It depends on interface specific requirements. For example, if the interface supports UCT_IFACE_FLAG_CONNECT_TO_IFACE only we don't need `dev_addr` (SELF specific).